src/openforms/prefill/base.py

```diff
@@ -65,6 +65,20 @@ def get_prefill_values(
         """
         raise NotImplementedError("You must implement the 'get_prefill_values' method.")
 
+    def verify_initial_data_ownership(
+        self, submission: Submission, prefill_options: dict
+    ) -> None:
+        """
+        Hook to check if the authenticated user is the owner of the object
+        referenced to by `initial_data_reference`
+
+        :param submission: an active :class:`Submission` instance
+        :param prefill_options: a dictionary containing the configuration options
+        """
+        raise NotImplementedError(
+            "You must implement the 'verify_initial_data_ownership' method."
+        )
```

Comment:
Discussed - if any error happens in a prefill plugin when initial_data_reference is populated, then we want this to bubble up as a HTTP 403 in the submission start. It's important that the end-user is confronted with this so that they can contact the municipality that something is wrong in their configuration, rather than continuing with the form without prefilled information.

The prefill module now suppresses a lot of exceptions - it logs them, but continues without prefill values.

Reply:
I've changed this, so now the user gets an error if prefill fails due to permissiondenied

![image](https://github.com/user-attachments/assets/d0e76ad0-b6ca-400a-b835-b8434152c9a4)

though I do not see the Submission show up in the admin, is this as expected?